src/utils.ts

```diff
@@ -23,6 +23,22 @@ export function generateKeypair(): KeyPair {
   }
 }
 
+export async function getPayload(
+  localPeer: PeerId,
+  staticPublicKey: bytes,
+  earlyData?: bytes,
+): Promise<bytes> {
+  const signedPayload = await signPayload(localPeer, getHandshakePayload(staticPublicKey));
+  const signedEarlyDataPayload = await signEarlyDataPayload(localPeer, earlyData || Buffer.alloc(0));
+
+  return await createHandshakePayload(
+    localPeer.marshalPubKey(),
+    localPeer.marshalPrivKey(),
```

Comment:
It looks like you could remove the `libp2pPrivateKey` argument from the `createHandshakePayload` function, as it's not being used.